app/views/shared/_micropost_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= form_for @micropost do |f| %>
+<%= render "shared/error_messages", object: f.object %>
+<div class="field">
+  <%= f.text_area :content, placeholder: t(".compose") %>
+</div>
+<%= f.submit t(".post"), class: "btn btn-primary" %>
+<span class="picture">
+  <%= f.file_field :picture, accept: "image/jpeg,image/gif,image/png" %>
+</span>
+<% end %>
+<script type="text/javascript">
+  $('#micropost_picture').bind('change', function() {
+    var size_in_megabytes = this.files[0].size/1024/1024;
+    if (size_in_megabytes > 5) {
+      alert('Maximum file size is 5MB. Please choose a smaller file.');
+    }
+  });
+</script>
```

Comment:
1. Use I18n
2. you shouldn't write javascript/css code into jsp file.
please, create js file, then write this code into it